### ISSUE TITLE
Danish Fungi 2020 - Performance Evaluation

### ISSUE THREAD

**Issue by picekl** [dqshuai/MetaFormer#2]
Dear All,

This is more a feature request than a bug; anyway, can you test/utilize your method on the DF20 dataset? We include much more metadata within the dataset, thus, the performance evaluation with regular ViT architecture might is interesting.

The link to the paper and web follows:
- https://sites.google.com/view/danish-fungi-dataset
- https://openaccess.thecvf.com/content/WACV2022/html/Picek_Danish_Fungi_2020_-_Not_Just_Another_Image_Recognition_Dataset_WACV_2022_paper.html

Best,
Lukas

**Comment by dqshuai** [comment]
I will try it！thanks. But, i can't open the web(https://sites.google.com/view/danish-fungi-datase),what should i do！

**Comment by picekl** [comment]
@dqshuai My bad! Missing t at the end! :)

Here is the correct link --> https://sites.google.com/view/danish-fungi-dataset

If you will need anything, please let me know via email.

PS: We are hosting a Kaggle competition. Might be a good idea to submit it. Looks like a winning method! 👍 
https://www.kaggle.com/c/fungiclef2022

Best,
Lukas

**Comment by iFighting** [comment]
> Dear All,
> 
> This is more a feature request than a bug; anyway, can you test/utilize your method on the DF20 dataset? We include much more metadata within the dataset, thus, the performance evaluation with regular ViT architecture might is interesting.
> 
> The link to the paper and web follows:
> 
> * https://sites.google.com/view/danish-fungi-datase
> * https://openaccess.thecvf.com/content/WACV2022/html/Picek_Danish_Fungi_2020_-_Not_Just_Another_Image_Recognition_Dataset_WACV_2022_paper.html
> 
> Best, Lukas

Thanks for your attention to our work，we will try it.
We also release our pretrain models, in case you want to try on other datasets

**Comment by dqshuai** [comment]
@picekl hi,I find the redirected url of "[Test Images][(max side size 300px)][~2.5Gb]" is same as [Test Images][~50Gb]. The file size is both 50gb. I'm guessing whether one of the links is wrong.

**Comment by picekl** [comment]
@dqshuai Good Point! It's fixed now.

FYI - The labels for the FungeCLEF2022 Test Data won't be available for a while. I would recommend you to split the train into (train/val) -- if you want to do some hyperparameter tuning -- and test on the validation set, which is more a public test set. Similar to validation on ImageNet-1k.
